docs/admin/architectures/index.md

```diff
@@ -171,3 +171,167 @@ Database:
   metadata.
 - Memory utilization averages at 40%.
 - `write_ops_count` between 6.7 and 8.4 operations per second.
+
+## Available reference architectures
+
+[Up to 1,000 users](1k-users.md)
+
+[Up to 2,000 users](2k-users.md)
+
+[Up to 3,000 users](3k-users.md)
+
+## Hardware recommendation
+
+### Control plane: coderd
+
+To ensure stability and reliability of the Coder control plane, it's essential
+to focus on node sizing, resource limits, and the number of replicas. We
+recommend referencing public cloud providers such as AWS, GCP, and Azure for
+guidance on optimal configurations. A reasonable approach involves using scaling
+formulas based on factors like CPU, memory, and the number of users.
+
+While the minimum requirements specify 1 CPU core and 2 GB of memory per
+`coderd` replica, it is recommended to allocate additional resources to ensure
+deployment stability.
+
+#### CPU and memory usage
+
+The memory consumption may increase with enabled agent stats collection by the
+Prometheus metrics aggregator (optional).
```

Comment:
```suggestion
Enabling [agent stats collection](link to docs) (optional) may increase memory consumption.
```

Reply:
Hm.. I don't see any relevant doc pages? Do you have something in mind?

Reply:
Yeah, CLI was what I meant.